klippy/extras/gcode_move.py

```diff
@@ -102,35 +105,47 @@ def get_status(self, eventtime=None):
             'extrude_factor': self.extrude_factor,
             'absolute_coordinates': self.absolute_coord,
             'absolute_extrude': self.absolute_extrude,
-            'homing_origin': self.Coord(*self.homing_position),
-            'position': self.Coord(*self.last_position),
-            'gcode_position': self.Coord(*move_position),
+            'homing_origin': self.Coord(*self.homing_position[:4]),
+            'position': self.Coord(*self.last_position[:4]),
+            'gcode_position': self.Coord(*move_position[:4]),
         }
     def reset_last_position(self):
         if self.is_printer_ready:
             self.last_position = self.position_with_transform()
+    def _update_extra_axes(self):
+        toolhead = self.printer.lookup_object('toolhead')
+        axis_map = {'X':0, 'Y': 1, 'Z': 2, 'E': 3}
```

Comment:
I believe 'E' would be returned by toolhead.get_extra_axes() below.

Reply:
Thanks for reviewing!

Right - the 'E' here is redundant.  I added it just to make it clear that the current gcode_move.py code isn't prepared for E to be at any other position that 4.  (In the future, we may want to treat extruders like any other kind of "extra axis".)